cluster/config.go

```diff
@@ -32,25 +33,17 @@ type hostConfigV9 struct {
 }
 
 func getS3Domain() string {
-	appDomain := "s3.localhost"
-	if os.Getenv(s3Domain) != "" {
-		appDomain = os.Getenv(s3Domain)
-	}
-	return appDomain
+	return env.Get(s3Domain, "s3.localhost")
 }
 
 func getM3ContainerImage() string {
-	concreteM3Image := "minio/m3:dev"
-	if os.Getenv(m3Image) != "" {
-		concreteM3Image = os.Getenv(m3Image)
-	}
-	return concreteM3Image
+	return env.Get(m3Image, "minio/m3:edge")
```

Comment:
I think this should be :dev cause all the yamls refer to that one.

Reply:
we don't have that container so its a bug to have :dev @cesnietor